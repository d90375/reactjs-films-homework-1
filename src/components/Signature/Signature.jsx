import React from "react";
import PropTypes from "prop-types";

import styles from "./signature.scss";

const Signature = props => {
  return <h1 className={styles.signature}>Hello, {props.name}</h1>;
};

Signature.propTypes = {
  name: PropTypes.string
};

export default Signature;
