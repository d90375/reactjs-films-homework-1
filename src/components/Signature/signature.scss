.nnnn {
    background-color: rgb(247, 124, 75);
}