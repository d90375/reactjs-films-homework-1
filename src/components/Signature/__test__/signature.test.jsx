import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Signature from "../../Signature";

test("Signature renders correctly", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Signature />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
