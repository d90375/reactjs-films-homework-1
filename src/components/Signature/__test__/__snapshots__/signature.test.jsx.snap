// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signature renders correctly 1`] = `
<h1
  className="signature"
>
  Hello, 
</h1>
`;
